src/main/kotlin/uk/gov/justice/digital/hmpps/dpssmoketest/service/QueueService.kt

```diff
@@ -93,7 +95,7 @@ class QueueService(
       .let { it.get().attributes()[QueueAttributeName.APPROXIMATE_NUMBER_OF_MESSAGES]?.toInt() ?: 0 }
 }
 
-data class AdditionalInformation(val nomsNumber: String, val details: String, val reason: String, val prisonId: String)
+data class AdditionalInformation(val nomsNumber: String, val details: String?, val reason: String, val prisonId: String)
```

Comment:
this can be `null` - see https://github.com/ministryofjustice/prison-offender-events/blob/450bee2f02c82da773e0617d444356c11e848e46/src/main/kotlin/uk/gov/justice/hmpps/offenderevents/services/ReleasePrisonerReasonCalculator.kt#L72